src/linker/runner.py

```diff
@@ -100,3 +99,14 @@ def get_environment_args(config: Config) -> List[str]:
             "only computing_environment 'local' and 'slurm' are supported; "
             f"provided {config.computing_environment}"
         )
+
+
+def get_num_jobs(config: Config) -> str:
+    if config.spark:
+        # If using spark, we need at least one job for the master,
+        # one job for each worker, and one job for the steps.
+        # It's more parallel to also have a job for each worker.
+        return str(2 * (config.spark_resources["num_workers"] + 1))
+    else:
+        # This might not be the right thing to do
```

Comment:
Is num_jobs needed at all? Can we just remove the `--jobs` argument altogether?

Reply:
you need to specify it for remote execution (though you can set it to `'unlimited`) When used locally, it's an alias for `--cores` so I'm fudging a bit by using it instead of specifying `--cores`
It might make more sense to just make this a CLI argument, similar to how it is done in `psimulate`.

Reply:
I think the default for `computing_environment: slurm` should be "unlimited" and the default for `computing_environment: local` should be the number of cores on the computer. If we want to let users override this default in the future, I'd like to do so in the `environment.yaml`.

Reply:
in that case, I think i can just use `--cores 'all' --jobs 'unlimited'` for now, which should work for both local and remote environments.